src/applications/disability-benefits/all-claims/constants.js

```diff
@@ -126,3 +126,8 @@ export const HOMELESS_HOUSING_TYPES = {
   anotherPerson: 'anotherPerson',
   other: 'other'
 };
+
+export const SERVICE_CONNECTION_TYPES = {
+  notServiceConnected: 'NOTSVCCON',
+  serviceConnected: 'SVCCONNCTED'
```

Comment:
The service-connected string is based on @SMLuthi 's digging into actual calls from staging
The not-service-connected string is straight from prod logs.